_posts/0000-01-02-saustin-bit.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
+Use the left arrow to go back!
```

Comment:
---
layout: slide
title: "Welcome to our second slide!"
---
Your text
Use the left arrow to go back!
With 5 lines of something new new new
new new
new new
new new new
yes sir